web/packages/teleport/src/Discover/Shared/AwsAccount/AwsAccount.tsx

```diff
@@ -151,7 +169,7 @@ export function ConnectAwsAccount() {
     }
 
     updateAgentMeta({
-      ...(agentMeta as DbMeta),
+      ...(agentMeta as NodeMeta),
```

Comment:
Which resource's flow we're in doesn't matter since all we're doing here is setting the `integration` field, though you're right that the typing isn't needed.